demos/python_demos/3d_segmentation_demo/3d_segmentation_demo.py

```diff
@@ -347,10 +381,30 @@ def main():
 
         im = 255 * (im - im.min())/(im.max() - im.min())
         color_seg_frame = np.zeros(im.shape, dtype=np.uint8)
-        for idx, c in enumerate(CLASSES_COLOR_MAP):
+        class_order = (0, 2, 1, 3)
```

Comment:
Why it's hardcoded here?

Reply:
@elenash please be patient. This item is in TODO list (see in PR description)